andela/hc-leo
# 157694805 Feature Help Center with FAQ’s, Help Videos & Live Chat

**What does this PR do?**
This PR adds help centre with FAQ’s, help videos, live chat. 

**Description of Task to be completed?**
The system documentation is limited to the README, the homepage with screenshots of the dashboard and the docs page the had documentation of the system. This feature provides sufficient content to users especially as assistants to the on boarding of new users to the platform. The feature integrates live chats and help videos in the FAQ's page.

**How should this be manually tested?**
- Since FAQs are dynamically added you need to load the db with initial content for FAQs. To do this, run ```./manage.py loaddata faqs.json```
- Navigate to ```/faq/``` to check the features added, ie, FAQ's, videos and live chat
- To add a new FAQ navigate to admin dashboad ```/admin/help/faq/```. Only users with admin rights can add an FAQ.


**What are relevant pivotal tracker stories?**
[#157694805](https://www.pivotaltracker.com/story/show/157694805)

**Screenshots**
> FAQ's
<img width="758" alt="screen shot 2018-06-19 at 21 14 17" src="https://user-images.githubusercontent.com/24602451/41616110-c8aa7aa4-7405-11e8-83bd-04051ead9490.png">

> live chat
<img width="574" alt="screen shot 2018-06-20 at 18 57 46" src="https://user-images.githubusercontent.com/24602451/41670031-e26c3728-74bb-11e8-8fe7-7ae871fcfeeb.png">


> help videos
<img width="770" alt="screen shot 2018-06-19 at 21 11 44" src="https://user-images.githubusercontent.com/24602451/41616150-e974ee7c-7405-11e8-9c44-0741fe367911.png">
